packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-json-metadata-visitor.ts

```diff
@@ -202,6 +203,11 @@ export class AppSyncJSONVisitor<
           isRequired: !field.isNullable,
           attributes: [],
         };
+
+        if (field.isListNullable !== undefined) {
+          fieldMeta.isArrayNullable = !field.isListNullable;
```

Comment:
With the rename, this should be changed to 
```suggestion
          fieldMeta.isArrayNullable = field.isListNullable;
```

Reply:
@yuth, Ah I did not think about this. I added a commit to fix this.